chapter_2/chapter_2.md

```diff
@@ -0,0 +1,47 @@
+# Chapter 2:
+
+* Bayesian is expensive and depends upon having a good model.
+
+* Principle of indifference: is to weight all conjectures equally. The structure of the model and the scientific context always provide information that allow us to do better than ignorance. 
```

Comment:
`is to weight`
Suggestion:
`it requires weighing`

Reply:
This still has issues because now it has a typo in "consists"

`* Principle of indifference: cosists in weighing all conjectures equally.`

As a general rule, use the suggestion I leave in the review unless there is a reason not to.